src/main/js/html.js

```diff
@@ -1374,51 +1398,52 @@ var backgroundColorAdjustSuffix = "BackgroundColorAdjust";
-                        attr[i] = attr[i].trim();
+                        if (attr[i].hasOwnProperty(i)) {
+                            attr[i] = attr[i].trim();
 
-                        if (attr[i] === "monospaceSerif") {
+                            if (attr[i] === "monospaceSerif") {
 
-                            rslt.push("Courier New");
-                            rslt.push('"Liberation Mono"');
-                            rslt.push("Courier");
-                            rslt.push("monospace");
+                                rslt.push("Courier New");
+                                rslt.push('"Liberation Mono"');
+                                rslt.push("Courier");
+                                rslt.push("monospace");
 
-                        } else if (attr[i] === "proportionalSansSerif" || attr[i] === "default") {
+                            } else if (attr[i] === "proportionalSansSerif" || attr[i] === "default") {
 
-                            rslt.push("Arial");
-                            rslt.push("Helvetica");
-                            rslt.push('"Liberation Sans"');
-                            rslt.push("sans-serif");
+                                rslt.push("Arial");
+                                rslt.push("Helvetica");
+                                rslt.push('"Liberation Sans"');
+                                rslt.push("sans-serif");
 
-                        } else if (attr[i] === "monospace") {
+                            } else if (attr[i] === "monospace") {
+                               
+                                rslt.push("monospace");
 
-                            rslt.push("monospace");
+                            } else if (attr[i] === "sansSerif") {
 
-                        } else if (attr[i] === "sansSerif") {
+                                rslt.push("sans-serif");
 
-                            rslt.push("sans-serif");
+                            } else if (attr[i] === "serif") {
 
-                        } else if (attr[i] === "serif") {
+                                rslt.push("serif");
 
-                            rslt.push("serif");
+                            } else if (attr[i] === "monospaceSansSerif") {
 
-                        } else if (attr[i] === "monospaceSansSerif") {
+                                rslt.push("Consolas");
+                                rslt.push("monospace");
 
-                            rslt.push("Consolas");
-                            rslt.push("monospace");
+                            } else if (attr[i] === "proportionalSerif") {
 
-                        } else if (attr[i] === "proportionalSerif") {
+                                rslt.push("serif");
 
-                            rslt.push("serif");
+                            } else {
 
-                        } else {
+                                rslt.push(attr[i]);
 
-                            rslt.push(attr[i]);
+                            }
```

Comment:
I'm surprised not to see @JibberJim 's code that postprocesses `rslt` to remove duplicates here. Can be done separately of course.

Reply:
I removed it from the branch as you'd already pushed it!

Reply:
Sorry, I'd forgotten I'd done that: of course it's #7. Thanks for the reminder! Rebasing will need to be judiciously applied depending on the order of merging, I guess, or we could just bring it in here too and close #7?